### PR TITLE
chaijs 1.10 allows to write lint-friendly tests

### PR DESCRIPTION
**Pull request by johanneswuerbach**
More details https://github.com/chaijs/chai/pull/297

### CODE REVIEW

File: tests/it-test.js [hunk 1]
```diff
@@ -67,13 +67,13 @@ describe('it', function() {
...
   it('skips tests with the .skip modifier', function() {
-    expect(skippedError).to.be.null;
+    expect(skippedError).to.be.null();
     var pendingSpec = Mocha.suite.suites.find(function(suite) {
       return suite.tests.find(function(test) {
-        return test.title === 'a skipped spec';
+        return test.title === 'is a skipped spec';
       });
     });
-    expect(pendingSpec).to.be.defined;
```
Comment (johanneswuerbach) [inline tests/it-test.js]: This was actually never doing the intended verification of  `pendingSpec`. `defined` is actually not a defined chai assertion and so this was testing for `undefined`. The new function syntax prevents those errors.

**Comment by rwjblue** [comment]
https://github.com/chaijs/chai/pull/297 was reverted in https://github.com/chaijs/chai/pull/306, so I don't think we can go with these changes....

**Comment by johanneswuerbach** [comment]
ok, I extracted the test fix https://github.com/switchfly/ember-mocha/issues/17 and will close this until chai released a new version, which might support this or now.

**Comment by dgeb** [comment]
Definitely looking forward to chai being more lint-friendly.

**Comment by Turbo87** [comment]
@dgeb apparently that's not going to happen, but it looks like [expect.js](https://github.com/Automattic/expect.js#api) is essentially a lint-friendly version of the same thing

**Pull request closed without merging**